[2025-07-15T23:30:11.038+0000] {taskinstance.py:1946} INFO - Dependencies not met for <TaskInstance: curated_market_dq.validate_dataframe manual__2025-07-15T23:27:48.249488+00:00 [failed]>, dependency 'Task Instance State' FAILED: Task is in the 'failed' state.
[2025-07-15T23:30:11.084+0000] {local_task_job_runner.py:160} INFO - Task is not able to be run
